src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,11 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> query = session.createQuery("FROM Movie", Movie.class);
+            return query.list();
```

Comment:
You can inline this string of code instead of the existing one. For instance, you can write 'return session.createQuery("from Movie", Movie.class).getResultList(). It will be a little simplified, won't it?